packages/fluentui/react-bindings/src/icons/createSvgIcon.tsx

```diff
@@ -0,0 +1,92 @@
+import * as React from 'react';
+// @ts-ignore
+import { ThemeContext } from 'react-fela';
+
+import { StylesContextValue } from '../styles/types';
+import useStyles from '../hooks/useStyles';
+import getUnhandledProps from '../utils/getUnhandledProps';
+import { SvgIconCreateFnParams, SvgIconProps } from './types';
+
+export const SvgIconClassName = 'ui-icon';
+export const SvgIconDisplayName = 'SvgIcon';
+
+export const SvgIconHandledProps: (keyof SvgIconProps)[] = [
```

Comment:
```suggestion
const svgIconHandledProps: (keyof SvgIconProps)[] = [
```

I don't think that we need to export this

Reply:
The same is used in the `SvgIcon` component, that's why we need to export it